src/main/resources/application.yml

```diff
@@ -32,6 +32,8 @@ data:
     baseUrl: ""
     username: ""
     password: ""
+    tokenUrl: ""
+    tokenMinValiditySeconds: 20
```

Comment:
Rather than sprinkling the default of `20` everywhere, why not default it in the `FhirConfig.java`?

Reply:
I may not understand how the application.yml works then... I thought this was where defaults were set?

Reply:
I think it would be weird to have a property missing in the yamls.